docs/windows.md

```diff
@@ -0,0 +1,195 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports to running on Windows worker Node, and setups an overlay network
+which uses VXLAN tunneling to route packets between nodes.
+
+One HNS Network with [Transparent mode](https://docs.microsoft.com/en-us/virtualization/windowscontainers/container-networking/network-drivers-topologies)
+is created on a Windows worker node, on which the [Open vSwitch Extension](http://docs.openvswitch.org/en/latest/intro/install/windows)
```

Comment:
node -> Node.
I personally feel no need to add "worker", but up to you.

Reply:
Sure, I think some of "worker"s can be removed.